array_string/[M]54.cpp

```diff
@@ -0,0 +1,29 @@
+// https://leetcode.com/problems/spiral-matrix/
+
+/* Bai nay hom truoc anh voi chu code roi, lan nay anh doc qua solution cua bon no de xem cach su dung direction.
```

Comment:
Thuc ra dung la cach nay code ngan' hon nhung hoi kho' hieu hon that. Luc pv chu code cai cach dung 4 vong for cung duoc.